charts/firefly/Chart.yaml

```diff
@@ -22,10 +22,10 @@ description: |
   ability to template additional multi-party or gateway namespaces.
 type: application
 appVersion: "1.3.0"
-version: "0.7.0"
+version: "0.8.0"
 
 maintainers:
-  - name: hfuss
+  - name: onelapahead
     email: hayden.fuss@kaleido.io
   - name: drewmarshburn
     email: drew.marshburn@kaleido.io
```

Comment:
Can you remove drew please

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/ceaa246d9a659fd704cbb5a3c55f0781d7c0c7fd